src/AzureDevOpsDemoBuilder/Views/Environment/CreateProject.cshtml

```diff
@@ -265,14 +265,14 @@
                                 </button>
                             </div>
                         </div>
-                        <div class="col-sm-12 form-group row d-none" id="txtRepoNamediv">
+                       <!-- <div class="col-sm-12 form-group row d-none" id="txtRepoNamediv">
```

Comment:
If possible, consider deleting instead of just commenting-out.

Reply:
removed text box which was used to accept repo name from user.